facter/facter.php

```diff
@@ -213,7 +213,11 @@ function Update()
 	if($resultarray) {
 		$id=$resultarray[0]['id'];
 		// Update SW type (OS)
-		$osrelease = $facter['operatingsystem'] . '%GSKIP%' . $facter['operatingsystem'] . ' V' . $facter['operatingsystemrelease'];
+		$os_release_lts = "";
+		if ($facter['operatingsystem'] == 'Ubuntu' && explode(".", $facter['operatingsystemrelease'])[0] % 2 == 0) {
```

Comment:
Would it be sufficient to link to the [Facter docs for it](https://docs.puppet.com/facter/latest/core_facts.html#operatingsystemrelease)?